modules/mobile/app/models/mobile/v0/adapters/facility_info.rb

```diff
@@ -4,29 +4,36 @@ module Mobile
   module V0
     module Adapters
       class FacilityInfo
-        def parse(facility, user, params)
-          user_location = params[:sort] == 'current' ? current_coords(params) : home_coords(user)
```

Comment:
Line 8 was the source of the bug. It only worked correctly when the sort method is current or the user has a home address. The issue wasn't previously uncovered because we didn't test with users who don't have addresses and because this endpoint is not in prod.